media/js/teach/smarton.js

```diff
@@ -0,0 +1,303 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+;(function($) {
+    'use strict';
+
+    var $window = $(window);
+    var $document = $(document);
+    var $body = $('body');
+    var $header = $('.page-header');
+    var $nav = $('.nav-steps');
+    var $firstSection = $('#ask');
+    var $firstStep = $('#nav-step-ask');
+    if ($header.length > 0) {
+        var headTop = $header.offset();
+        var headHeight = $header.height();
+    }
+
+    if (window.isFirefox()) {
+        $body.addClass('is-firefox');
+    } else {
+        $body.addClass('not-firefox');
+    }
+
+    // Sticky navigation
+    var fixed = false;
+    var didScroll = false;
+
+    $window.scroll(function() {
+        didScroll = true;
+    });
+
+    if ($header.length > 0) {
+        $(document).ready(function() {
+            var scrollTop = $window.scrollTop();
+            if (scrollTop >= headTop.top) {
+                didScroll = true;
+            }
+        });
+    }
+
+    function adjustScrollbar() {
+        if (didScroll) {
+            didScroll = false;
+            var scrollTop = $window.scrollTop();
+            if ($header.length > 0) {
+                if( scrollTop >= headTop.top ) {
+                    if(!fixed) {
```

Comment:
Nit: conditionals in this file could do with some tidy up in places, e.g. spaces after `if`, spaces before opening brackets `{` etc. Obviously this is less important than other things, so feel free to skip it if time is crunched.